subiquity/controllers/filesystem.py

```diff
@@ -260,46 +236,55 @@ def add_raid_dev(self, result):
         self.model.add_raid_device(result)
         self.signal.prev_signal()
 
-    def add_first_gpt_partition(self, *args, **kwargs):
-        self.ui.set_body(DummyView(self.signal))
-
     @view
-    def create_swap_entire_device(self, disk):
-        log.debug('create_swap_entire_device')
-        log.debug("formatting whole {}".format(disk))
+    def format_entire(self, disk):
+        log.debug("format_entire {}".format(disk))
+        header = ("Format and/or mount %s"%(disk.path))
```

Comment:
Another place where `.format()` is maybe more consistent with the surrounding code?